src/admin/components/website/modalParts/WebsiteFeedbackEmailOverride.tsx

```diff
@@ -90,44 +106,63 @@ class WebsiteFeedbackEmailOverride extends React.Component <any, any> {
 		});
 	}
 
-	render () {
-		const feedbackEmailOverride = this.props.feedbackEmailOverride.map((email, index) => {
-			return (
+	buildList () {
+		return this.props.feedbackEmailOverride.map((email, index) => (
 				<li key={index + '-email-override'} className="website-feedback-email-override-list-item">
 					{email}
 					<i className="fa fa-times" onClick={this.onDelete.bind(this, index)}/>
-				</li>);
-		});
+				</li>));
+	}
+
+	render () {
+		const {value, touched} = this.state;
+		const {settings: {feedback: feedbackOverrideStatus = false}} = this.props;
 		return (
-			<div className="medium-12 columns">
+			<div className="medium-12 columns website-feedback-email-override-container overrides-container">
 				<fieldset className="text">
-					<label htmlFor="feedback-email-override">
-						<b>Feedback email override</b><br/>
-						Emails to get notifications about feedbacks for articles if needed
-						(instead of article authors emails)
-					</label>
-					<input
-						id="feedback-email-override" type="text" className="small-12 medium-12"
-						value={this.state.value}
-						onChange={this.onEdit}
-						onKeyPress={this.onKeyPress} onBlur={this.onSubmit}
-					/>
-					<InputError
-						errorText={this.checkValidation()}
-						touchedField={this.state.touched}
-					/>
-					<ul className="website-feedback-email-override-list">
-						{feedbackEmailOverride}
-					</ul>
+					<div className="row">
+						<div className="small-12 columns hide-for-medium columns override-status-control">
+							<SwitchBox
+								classes={`switch round tiny`} ID={'feedback-email-override-status-small'}
+								checked={feedbackOverrideStatus} onChange={this.onToggleCheck}
+							/>
+						</div>
+						<div className="small-12 medium-9 large-10 columns">
+							<LabeledInput
+								onSubmit={this.onSubmit} errorText={this.checkValidation()}
+								onEdit={this.onEdit} value={value}	touched={touched}
+								label={<span>
+										<b>Feedback email override</b><br/>
+										Emails to get notifications about feedbacks for articles if needed
+										(instead of article authors emails)
+									</span>}
```

Comment:
Don't think this is normal way to set the html

Reply:
Any suggestions are welcome, found no good way before

Reply:
https://reactjs.org/docs/dom-elements.html

Reply:
nonono, I don't trust this thing at all. Dangerous is dangerous

Reply:
I meant another thing.
I think you can set a variable for label content, not writing this inside of the component.

Reply:
Also, you set the same label in WebsiteOverrideSettingsContentSection.tsx with text about feedback email. What if you want to customize it? 
Because there is a text about email override inside it!

Reply:
For the WebsiteOverrideSettingsContentSection - it is a component that should replace existing one you are comenting on. So while refactor this message will replace exiting static one for description for all similar components